packages/dx-react-chart-bootstrap4/src/templates/tooltip/arrow.jsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+export const Arrow = ({ arrowProps, className }) => (
+  <div className={classNames('arrow', className)} {...arrowProps} />
+);
+Arrow.propTypes = {
+  arrowProps: PropTypes.object.isRequired,
```

Comment:
This property is not documented. Should we do that?

- This component doesn't spread rest properties.
- According to the documentation, this component has property `placement`.

Reply:
- The `arrowProps` property is used due to features of `react-popper`. I think it should not be documented.
- I add `placement` property.
- I add rest properties.